h2o-core/src/main/java/org/eclipse/jetty/plus/jaas/spi/LdapLoginModule.java

```diff
@@ -0,0 +1,8 @@
+package org.eclipse.jetty.plus.jaas.spi;
```

Comment:
I suppose this class should not be in the core

Reply:
somehow I forgot to delete it, thanks for pointing this out.
I also realized that `h2o-core` still depends on jetty; this is too bad, now I removed it and am struggling with making the build functional again, I may need your help tomorrow as I seem to have encountered another weird butterfly effect (`Task :h2o-bindings:runGenerateRESTAPIBindingsSrc FAILED`)